src/app/pages/jails/jail-list/jail-list.component.ts

```diff
@@ -146,16 +146,17 @@ export class JailListComponent implements OnInit {
     },
   ];
 
+  public tooltipMsg: any = T("Choose a pool where the iocage jail manager \
```

Comment:
this tooltip message should probably go in a helper file rather than being in the component itself